benches/select_market_data.rs

```diff
@@ -41,21 +41,21 @@ async fn prepare_data() {
     client
         .query(
             r#"
-            CREATE TABLE IF NOT EXISTS l2_book_log
-            (
-                `instrument_id` UInt32 CODEC(T64, Default),
-                `received_time` DateTime64(9) CODEC(DoubleDelta, Default),
-                `exchange_time` Nullable(DateTime64(9)) CODEC(DoubleDelta, Default),
-                `sequence_no` Nullable(UInt64) CODEC(DoubleDelta, Default),
-                `trace_id` UInt64 CODEC(DoubleDelta, Default),
-                `side` Enum8('Bid' = 0, 'Ask' = 1),
-                `price` Float64,
-                `amount` Float64,
-                `is_eot` Bool
-            )
-            ENGINE = MergeTree
-            PRIMARY KEY (instrument_id, received_time)
-        "#,
+                CREATE TABLE IF NOT EXISTS l2_book_log
+                (
+                    `instrument_id` UInt32                      CODEC(T64,         Default),
+                    `received_time` DateTime64(9)               CODEC(DoubleDelta, Default),
+                    `exchange_time` Nullable(DateTime64(9))     CODEC(DoubleDelta, Default),
+                    `sequence_no`   Nullable(UInt64)            CODEC(DoubleDelta, Default),
+                    `trace_id`      UInt64                      CODEC(DoubleDelta, Default),
+                    `side`          Enum8('Bid' = 0, 'Ask' = 1),
+                    `price`         Float64,
+                    `amount`        Float64,
+                    `is_eot`        Bool
+                )
+                ENGINE = MergeTree
+                PRIMARY KEY (instrument_id, received_time)
+            "#,
```

Comment:
Using raw string literals is also something idiomatic to SQLx, though strictly speaking this is only necessary for when you need to quote identifiers as it avoids the need for escapes: https://docs.rs/sqlx/latest/sqlx/macro.query.html#type-overrides-output-columns

Since it appears ClickHouse uses backticks instead of double quotes for quoted identifiers, there's not much benefit to using raw string literals here.